src/redux/actions/index.js

```diff
@@ -17,3 +29,51 @@ export const fetchImagesSuccess = imageURL => ({
 export const requestImages = () => ({
   type: REQUEST_IMAGES,
 })
+
+// Images Gallery
+export const fetchImagesGallerySuccess = imagesItems => ({
+  type: FETCH_IMAGES_GALLERY_SUCCESS,
+  payload: {
+    imagesItems,
+  },
+})
+
+export const requestImagesGallery = () => ({
+  type: REQUEST_IMAGES_GALLERY,
+})
+
+// Search
+export const setSearchWord = searchText => ({
+  type: SET_SEARCH_WORD,
+  payload: {
+    searchText,
+  },
+})
+
+// Pagination
+export const setCurrentPage = page => ({
```

Comment:
what type is page ? is it a number ? like 0, 1, 2, 3 etc... ?  if that's a number you better call it:
**pageIndex** instead of **page**

always try to be more specific when you name variables. if you say **page** I imagine myself an action that receives a full HTML page as a parameter.
If you say **pageIndex** - everything is clear